TWEET.md

```diff
@@ -0,0 +1,3 @@
+This release improves how we deal with custom scalars. Instead of being
+global they are now scoped to the current schema. You can also override
+the built in scalars with your own custom implementation.
```

Comment:
```suggestion
🆕 Release $version is out! Thanks to @jkimbo for this great new feature! 
We are now able to override the default scalars, so you don't have to stick
with the defaults provided by us 😊

Get it here 👉 $release_url

---

This release improves how we deal with custom scalars. Instead of being
global they are now scoped to the current schema. You can also override
the built in scalars with your own custom implementation.
```